apps/etterlatte-behandling/src/main/kotlin/tilgangsstyring/Tilgangsstyring.kt

```diff
@@ -36,6 +37,8 @@ class PluginConfiguration {
     -> Boolean = { _, _ -> false }
     var harTilgangTilOppgave: (oppgaveId: String, saksbehandlerMedRoller: SaksbehandlerMedRoller)
     -> Boolean = { _, _ -> false }
+    var harTilgangTilKlage: (klageId: String, saksbehandlerMedRoller: SaksbehandlerMedRoller)
```

Comment:
Burde den tatt imot en `UUID` for å sikre type på `klageId` ?

Reply:
Jeg skal se på det, det gjelder forsåvidt de andre typesjekkene på UUID og bør dermed endres felles :)